plugins/inventory/nb_inventory.py

```diff
@@ -586,7 +586,17 @@ def extract_primary_ip6(self, host):
             return
 
     def extract_tags(self, host):
-        return host["tags"]
+        try:
+            tagobj = host["tags"][0]
+            # Check the format of the first element in the "tags" array.
+            # Netbox v2.9+ uses dicts, so return an array of just tags' names.
+            if isinstance(tagobj, dict):
+                return [sub["name"] for sub in host["tags"]]
+            # Netbox v2.8 and below uses strings, so return the plain array.
+            elif isinstance(tagobj, str):
+                return host["tags"]
+        except Exception:
+            return
```

Comment:
This will return None if there are no tags.  Is that OK?  Would `return []` be safer?

Reply:
Good catch. Fixed.